projects/1.19/assets/domum-ornamentum/domum_ornamentum/lang/zh_cn.json

```
@@ -0,0 +1,137 @@
+  "block.domum_ornamentum.gray_floating_carpet": "灰色浮空地毯",
+  "block.domum_ornamentum.green_floating_carpet": "绿色浮空地毯",
+  "block.domum_ornamentum.light_blue_floating_carpet": "淡蓝色浮空地毯",
+  "block.domum_ornamentum.light_gray_floating_carpet": "淡灰色浮空地毯",
+  "block.domum_ornamentum.lime_floating_carpet": "黄绿色浮空地毯",
+  "block.domum_ornamentum.magenta_floating_carpet": "品红色浮空地毯",
+  "block.domum_ornamentum.orange_floating_carpet": "橙色浮空地毯",
+  "block.domum_ornamentum.pink_floating_carpet": "粉红色浮空地毯",
+  "block.domum_ornamentum.purple_floating_carpet": "紫色浮空地毯",
+  "block.domum_ornamentum.red_floating_carpet": "红色浮空地毯",
+  "block.domum_ornamentum.sand_bricks": "沙子砖块",
+  "block.domum_ornamentum.sand_stone_bricks": "沙子石砖",
+  "block.domum_ornamentum.white_floating_carpet": "白色浮空地毯",
+  "block.domum_ornamentum.yellow_floating_carpet": "黄色浮空地毯",
+  "domum_ornamentum.architectscutter": "建筑师切割机",
+  "domum_ornamentum.block.format": "材料：%s",
+  "domum_ornamentum.blockpillar.name.format": "圆形%s柱子",
+  "domum_ornamentum.blockypillar.name.format": "体素%s柱子",
+  "domum_ornamentum.door.block.format": "材料：%s",
+  "domum_ornamentum.door.name.format": "%s门",
+  "domum_ornamentum.door.type.format": "变种: %s",
+  "domum_ornamentum.door.type.name.full": "填满",
+  "domum_ornamentum.door.type.name.port.manteau": "斗篷形",
+  "domum_ornamentum.door.type.name.vertically.striped": "垂直条纹",
+  "domum_ornamentum.door.type.name.waffle": "田字",
+  "domum_ornamentum.extra.name.format": "%s%s附加装饰方块",
+  "domum_ornamentum.fancydoor.center.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancydoor.center.header": "中心：",
+  "domum_ornamentum.fancydoor.frame.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancydoor.frame.header": "边框：",
+  "domum_ornamentum.fancydoor.name.format": "精致%s门",
+  "domum_ornamentum.fancydoor.type.format": "变种：%s",
+  "domum_ornamentum.fancydoor.type.name.creeper": "苦力怕",
+  "domum_ornamentum.fancydoor.type.name.full": "填满",
+  "domum_ornamentum.fancytrapdoor.center.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancytrapdoor.center.header": "中心：",
+  "domum_ornamentum.fancytrapdoor.frame.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancytrapdoor.frame.header": "边框：",
+  "domum_ornamentum.fancytrapdoor.name.format": "精致%s活板门",
+  "domum_ornamentum.fancytrapdoor.type.format": "变种: %s",
+  "domum_ornamentum.fancytrapdoor.type.name.creeper": "苦力怕",
+  "domum_ornamentum.fancytrapdoor.type.name.full": "填满",
+  "domum_ornamentum.fence-gate.name.format": "%s栅栏门",
+  "domum_ornamentum.fence.name.format": "%s栅栏",
+  "domum_ornamentum.light.center.block.format": "  - 材料: %s",
+  "domum_ornamentum.light.center.header": "中心:",
+  "domum_ornamentum.light.frame.block.format": "  - 材料: %s",
+  "domum_ornamentum.light.frame.header": "边框:",
+  "domum_ornamentum.light.frame.name.format": "镶边%s",
+  "domum_ornamentum.light.frame.type.crossed_light": "十字",
+  "domum_ornamentum.light.frame.type.fancy_light": "精致",
+  "domum_ornamentum.light.frame.type.format": "  - 样式: %s",
```

```suggestion
  "domum_ornamentum.light.frame.type.format": "  - 样式：%s",
```
所有标点符号如无必要或惯例均使用中文全角标点